tests/integration/dataframe/test_dataframe.py

```diff
@@ -1,49 +1,76 @@
 # -*- coding: utf-8 -*-
-from typing import Iterator
+from datetime import datetime
+from typing import List
 
 import pytest  # type: ignore
+from nisystemlink.clients.core import ApiException
 from nisystemlink.clients.dataframe import DataFrameClient
 from nisystemlink.clients.dataframe import models
 
 
-@pytest.mark.enterprise
-@pytest.mark.integration
-class TestDataFrame:
-    @pytest.fixture
-    def client(self, enterprise_config):
-        return DataFrameClient(enterprise_config)
+@pytest.fixture(scope="class")
+def client(enterprise_config):
+    """Fixture to create a DataFrameClient instance."""
+    return DataFrameClient(enterprise_config)
 
-    @pytest.fixture
-    def table_id(self, client: DataFrameClient) -> Iterator[str]:
-        id = client.create_table(
-            models.CreateTableRequest(
-                columns=[
-                    models.Column(
-                        name="time",
-                        data_type=models.DataType.Timestamp,
-                        column_type=models.ColumnType.Index,
-                        properties={"cat": "meow"},
-                    ),
-                    models.Column(name="value", data_type=models.DataType.Int32),
-                ],
-                name="Python API test table (delete me)",
-                properties={"dog": "woof"},
+
+@pytest.fixture(scope="class")
+def create_table(client: DataFrameClient):
+    """Fixture to return a factory that creates tables."""
+    tables = []
+
+    def _create_table(table: models.CreateTableRequest) -> str:
+        id = client.create_table(table)
+        print(f"Created table {id}")
+        tables.append(id)
+        return id
+
+    yield _create_table
+
+    for id in tables:
+        client.delete_table(id)
+        print(f"Deleted table {id}")
```

Comment:
We may want to leave a note to change this to use the multi-delete route when we have it.